### PR TITLE
Adding additional best practice policies

### PR DESCRIPTION
**Pull request by nsagark**
Adding additional best practice policies

### CODE REVIEW

**Comment by patelrit** [comment]
Some comments:
- Please follow the same naming convention as other policies - for validate policies, please follow terms such as 'restrict', 'prevent', 'disallow' etc
- Please do not include any mutate or generate policies in the default policy sets as these policy sets will be included in default sets. We can create different policy sets for mutate and generate policies which can be available as optional.

**Comment by nsagark** [comment]
I have renamed the validate policies and removed the mutate/generate policies. Please review

File: best-practices/disallow_secrets_from_env_vars/disallow_secrets_from_env_vars.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: secrets-not-from-env-vars
+  annotations:
+    policies.kyverno.io/title: Disallow Secrets from Env Vars
+    policies.kyverno.io/category: Sample, EKS Best Practices
```
Comment (anusha94) [inline best-practices/disallow_secrets_from_env_vars/disallow_secrets_from_env_vars.yaml:7]: Do not use Sample as a category

File: best-practices/prevent_orphan_pods/prevent_orphan_pods.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: prevent-naked-pods
+  annotations:
+    policies.kyverno.io/title: Prevent Naked Pods
+    pod-policies.kyverno.io/autogen-controllers: none
+    policies.kyverno.io/category: Other, EKS Best Practices
```
Comment (anusha94) [inline best-practices/prevent_orphan_pods/prevent_orphan_pods.yaml:8]: Do not use Other as a category

**Review by anusha94 — CHANGES_REQUESTED** [review]
There are other occurrences of `Other` and `Sample` in category annotation. Please remove those.

**Pull request closed without merging**